integration/docker-compose.otel.yml

```diff
@@ -26,7 +26,7 @@ services:
   prometheus:
     image: prom/prometheus:v2.36.2
     volumes:
-      - ./otel/prometheus.yaml:/etc/prometheus/prometheus.yml
+      - ./otel/prometheus.yaml:/etc/prometheus/prometheus.yml:z
```

Comment:
What does the `:z` do here?

Reply:
That's SELinux permission for that file: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label

Reply:
Oh, I see you mentioned this in the commit message, so all good and I understand it's SELinux related.